<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Switches_box</class>
 <widget class="QDialog" name="Switches_box">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>D3</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="cb_D3_1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D3_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D3_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D3_4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>D8</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QCheckBox" name="cb_D8_1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D8_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D8_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D8_4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>D5</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="cb_D5_1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D5_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D5_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D5_4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>D1</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="cb_D1_1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D1_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D1_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D1_4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>D2</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QCheckBox" name="cb_D2_1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D2_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D2_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D2_4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>D9</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QCheckBox" name="cb_D9_1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D9_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D9_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D9_4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>D4</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QCheckBox" name="cb_D4_1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D4_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D4_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D4_4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>D6</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="cb_D6_1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D6_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D6_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D6_4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>D7</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="cb_D7_1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D7_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D7_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_D7_4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_D1_1</tabstop>
  <tabstop>cb_D1_2</tabstop>
  <tabstop>cb_D1_3</tabstop>
  <tabstop>cb_D1_4</tabstop>
  <tabstop>cb_D2_1</tabstop>
  <tabstop>cb_D2_2</tabstop>
  <tabstop>cb_D2_3</tabstop>
  <tabstop>cb_D2_4</tabstop>
  <tabstop>cb_D3_1</tabstop>
  <tabstop>cb_D3_2</tabstop>
  <tabstop>cb_D3_3</tabstop>
  <tabstop>cb_D3_4</tabstop>
  <tabstop>cb_D4_1</tabstop>
  <tabstop>cb_D4_2</tabstop>
  <tabstop>cb_D4_3</tabstop>
  <tabstop>cb_D4_4</tabstop>
  <tabstop>cb_D5_1</tabstop>
  <tabstop>cb_D5_2</tabstop>
  <tabstop>cb_D5_3</tabstop>
  <tabstop>cb_D5_4</tabstop>
  <tabstop>cb_D6_1</tabstop>
  <tabstop>cb_D6_2</tabstop>
  <tabstop>cb_D6_3</tabstop>
  <tabstop>cb_D6_4</tabstop>
  <tabstop>cb_D7_1</tabstop>
  <tabstop>cb_D7_2</tabstop>
  <tabstop>cb_D7_3</tabstop>
  <tabstop>cb_D7_4</tabstop>
  <tabstop>cb_D8_1</tabstop>
  <tabstop>cb_D8_2</tabstop>
  <tabstop>cb_D8_3</tabstop>
  <tabstop>cb_D8_4</tabstop>
  <tabstop>cb_D9_1</tabstop>
  <tabstop>cb_D9_2</tabstop>
  <tabstop>cb_D9_3</tabstop>
  <tabstop>cb_D9_4</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Switches_box</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Switches_box</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
